src/wallet/rpcwallet.cpp

```diff
@@ -3874,9 +3874,10 @@ static UniValue listlabels(const JSONRPCRequest& request)
 static RPCHelpMan send()
 {
     return RPCHelpMan{"send",
+        "\nEXPERIMENTAL warning: this call may be changed in future releases.\n"
         "\nSend a transaction.\n",
         {
-            {"outputs", RPCArg::Type::ARR, RPCArg::Optional::NO, "a json array with outputs (key-value pairs), where none of the keys are duplicated.\n"
+            {"outputs", RPCArg::Type::ARR, RPCArg::Optional::NO, "A JSON array with outputs (key-value pairs), where none of the keys are duplicated.\n"
```

Comment:
```suggestion
            {"outputs", RPCArg::Type::ARR, RPCArg::Optional::NO, "Outputs (key-value pairs), where none of the keys are duplicated.\n"
```

The whole point of RPCHelpMan is to deal with types and formatting, so that the documentation strings can be minimal.

If you repeat the types, it will be rendered twice:

```
1. outputs                               (json array, required) a json array with outputs (key-value pairs), where none of the keys are duplicated.

Reply:
Will fix if there's other issues (or just later)